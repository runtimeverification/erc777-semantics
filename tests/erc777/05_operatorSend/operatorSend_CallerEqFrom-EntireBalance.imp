test.preamble();

// Testing operatorSend(From,To,Value,"","") when From = caller.
// Entire balance sent.

test.operatorSend(7, 4, 70, "", "")  // balance of caller is 70
