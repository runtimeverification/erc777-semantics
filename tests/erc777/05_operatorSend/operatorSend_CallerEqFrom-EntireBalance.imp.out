7 is sending 70 from 7 to 4 (data:""),  (operatorData:"")
Balance of 7 is 0
Balance of 4 is 110
